### ISSUE TITLE
Selecting Debug configuration does not result in symbol files in nuget packages

### ISSUE THREAD

**Issue by mrj001** [AvaloniaUI/Avalonia#4927]
Building as follows:
```
$ git clean -xdf
$ ./build.sh --target Package --skip CompileHtmlPreviewer --configuration Debug
```
Then
`$ find . -name \*.symbols.nupkg`
produces no output.  So no symbol packages were produced.

In my test project folder, I did `rm -r packages` and then recreated that nuget repo via nuget init from the build-intermediate/nuget folder.

Then, I did `rm -r bin obj`, followed by `dotnet build`.  The 'bin/Debug/net5.0/osx-x64' output folder does not contain any Avalonia .pdb files, just the one pdb file for my test project.

How does one create debuggable nuget packages for Avalonia?  I believe the above should have done so.

**Comment by kekekeks** [comment]
Usually we use sourcelink, but one of the recent PRs has broken pdb generation. Should be fixed shortly.

**Comment by MrJul** [comment]
Fixed by #12216 (with .snupkg symbol packages)
